Yoast/wordpress-seo
# P1-102 add open post settings button to sidebar

## Context


* We want to be able to open the `PostSettingsModal` from the sidebar.

## Summary


This PR can be summarized in the following changelog entry:

* Adds a button to the sidebar that opens the `PostsettingsModal` when clicked.

## Relevant technical choices:

*

## Test instructions

This PR can be tested by following these steps:
*After https://github.com/Yoast/wordpress-seo/pull/15794 is merged*
1. Click the button
2. `postSettingsModal.isOpen` should change in the store to `true` (use Redux dev-tools to see).

## UI changes

## Documentation

## Quality assurance


Fixes P1-102
